modules/ipi-install-replacing-a-bare-metal-control-plane-node.adoc

```diff
@@ -39,7 +39,7 @@ $ oc get clusteroperator baremetal
 [source,terminal]
 ----
 NAME        VERSION   AVAILABLE   PROGRESSING   DEGRADED   SINCE   MESSAGE
-baremetal   4.12.0   True        False         False      3d15h
+baremetal   4.15.0   True        False         False      3d15h
```

Comment:
```suggestion
baremetal   {product-version}   True        False         False      3d15h
```
You can use the attribute to replace the hard-coded version, but it will just state major.minor (as in "4.14" and so on) as the attribute does not include the z-stream reference.

Reply:
If you still want to merge this, replace the `4.15.0` with {product-version}.